packages/cli/src/commands/generate.ts

```diff
@@ -116,6 +117,10 @@ export class Generate extends Command {
       char: "d",
       description: "Show what files will be created without writing them to disk",
     }),
+    templateDir: flags.string({
+      char: "t",
+      description: "Specify a custom template directory",
+    }),
```

Comment:
@beerose lets make this an option in blitz.config.ts instead of a CLI flag, so we don't have to pass this flag every time.

Other than that, it looks great